src/map/battle.cpp

```diff
@@ -3016,6 +3016,7 @@ static bool is_attack_critical(struct Damage* wd, struct block_list *src, struct
 #ifdef RENEWAL
 			case ASC_BREAKER:
 #endif
+			case IG_RADIANT_SPEAR:
```

Comment:
```suggestion
```
crit chance should not be halved

Reply:
@datawulf are you sure about that?
From the skill description: "The effectiveness of critical modifier is applied by half."

Reply:
> @datawulf are you sure about that? From the skill description: "The effectiveness of critical modifier is applied by half."

He is right, what you mentioned is the default critical factor from kRO, that the critical damages is halved